presto-native-sidecar-plugin/src/test/java/com/facebook/presto/sidecar/TestNativeSidecarPluginSystemPropertyProvider.java

```diff
@@ -72,14 +72,7 @@ public void testShowSession()
     @Test
     public void testSetJavaWorkerSessionProperty()
```

Comment:
need a test that when exclude-invalid-session-properties is false that the session properties are visible. (and make sure we have true and false tests both for java and cpp.